src/uffd_handler/Cargo.toml

```diff
@@ -10,7 +10,7 @@ license = "Apache-2.0"
 
```

Comment:
nit: typo in the commit message description.
Also, consider using https://www.conventionalcommits.org/en/v1.0.0/#summary for being explicit about the purpose of each commit.